src/osha/oira/client/browser/client.py

```diff
@@ -229,8 +229,11 @@ def results(self):
             .filter(Account.loginname.in_(recipients))
             .distinct()
         )
+        country = path.partition("/")[0]
```

Comment:
How can this work?
The country is defined outside the list comprehension...

Reply:
LOL `flake8` did not complain on this because we do an `from os import path` at the top of the file :D

Reply:
Wow :facepalm: I've called it `zodb_path` now and fixed it.